systemtest/src/main/java/io/strimzi/systemtest/utils/ClientUtils.java

```diff
@@ -43,11 +43,18 @@ public static void waitUntilClientReceivedMessagesTls(KafkaClientOperations kafk
     }
 
     public static void waitTillContinuousClientsFinish(String producerName, String consumerName, String namespace, int messageCount) {
-        long timeout = (long) messageCount * 1000;
+        long timeout = (long) messageCount * 3000;
```

Comment:
Why do you change this ? I don't see usage in your changes. Does it cause problems in nightly builds?

Reply:
Yeah sorry! I used this method in my test update, but then I removed it, good catch :)